.travis.yml

```diff
@@ -3,11 +3,17 @@ cache: pip
 dist: xenial
 services:
   - docker
-addons:
-apt:
-  packages:
-    - python-mysqldb
 install:
+  # Microsoft ODBC driver installation.
+  - sudo bash -c "curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -"
+  - sudo bash -c "curl https://packages.microsoft.com/config/ubuntu/$(lsb_release -r -s)/prod.list > /etc/apt/sources.list.d/mssql-release.list"
```

Comment:
Can you reconcile this with the best-practices recommendations at https://docs.travis-ci.com/user/installing-dependencies/ ?